l10n_it_codici_carica/models/codice_carica.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Alessandro Camilli - Openforce
+# Copyright 2019 Stefano Consolaro (Associazione PNLUG - Gruppo Odoo)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class CodiceCarica(models.Model):
+    _name = 'codice.carica'
+    _description = 'Role code'
```

Comment:
In inglese, nei nomi o nei titoli,  è prassi usare la maiuscola nelle lettere iniziali delle parole.
→ Role Code